docs/custom-domain/README.md

```diff
@@ -91,6 +106,12 @@ For each domain, its TLS private key and public certificate pair must be added t
 
 For greater isolation, we recommend that you put each certificate in its own key vault and set its access policy independently. The registry should always have access to the key vault secrets.
 
+### Certificate updates and rotation
+
+The registry configures each custom domain certificate without referencing a specific version in Azure Key Vault. With this configuration, the registry regularly checks the key vault and automatically uses the latest certificate version there for its operations.
```

Comment:
This is configurable. Customers have the option to use a versioned secret ID as well if they do not want auto cert rotation.

Reply:
Updated - thanks @aviral26